store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1623,13 +1629,41 @@ private boolean execute(String args[]) throws ServiceException, ArgException, IO
             break;
         case MODIFY_HAB_GROUP_SENIORITY:
             modifyHabGroupSeniority(args);
+        case ADD_HAB_GROUP_MEMBER:
+            doAddMember(args);
+            break;
+        case DELETE_HAB_GROUP:
+            doDeleteDistributionList(args);
+            break;
+        case REMOVE_HAB_GROUP_MEMBER:
+            doRemoveMember(args);
+            break;
+        case GET_HAB_GROUP_MEMBERS:
+            doGetHABGroupMembers(args);
             break;
         default:
             return false;
         }
         return true;
     }
 
+    private void doAddMember(String[] args) throws ServiceException {
```

Comment:
Do we need to add arg length check and usage() call for each of these?

Reply:
It's already handled. 
zimbra@sprasad:~$ zmprov adlm 
usage:  addDistributionListMember(adlm) {list@domain|id} {member@domain}+
For general help, type : zmprov --help
zimbra@sprasad:~$ zmprov adlm f02ca108-14d5-4f23-a0c4-53d92c8bebdd
usage:  addDistributionListMember(adlm) {list@domain|id} {member@domain}+
For general help, type : zmprov --help